packages/iscp/assets.go

```diff
@@ -175,6 +197,47 @@ func (a *Assets) Equals(b *Assets) bool {
 	return true
 }
 
+// FitsTheBudget checks if:
+// - 'a' has all non-negative values. Negative values is error
+// - 'a' all values are <= of corresponding values of 'b'. It means, we can debit 'a' from 'b' without overrun of funds
+func (a *Assets) FitsTheBudget(budget *Assets) (bool, error) {
+	if a == budget {
+		return true, nil
+	}
+	if a.IsEmpty() {
+		return true, nil
```

Comment:
```suggestion
		return false, nil
```

should be false, right?